packages/shared/src/utils/contract/contract.ts

```diff
@@ -1,51 +1,73 @@
 import moment from 'moment';
```

Comment:
You don't have to review this file! I just merged it with my branch to fix TS errors for helpers